xchange-cexio/src/main/java/info/bitrich/xchangestream/cexio/CexioStreamingService.java

```diff
@@ -0,0 +1,134 @@
+package info.bitrich.xchangestream.cexio;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import info.bitrich.xchangestream.cexio.dto.*;
+import info.bitrich.xchangestream.service.netty.JsonNettyStreamingService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+public class CexioStreamingService extends JsonNettyStreamingService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CexioStreamingService.class);
+
+    public static final String CONNECTED = "connected";
+    public static final String AUTH = "auth";
+    public static final String PING = "ping";
+    public static final String PONG = "pong";
+    public static final String ORDER = "order";
+
+    private String apiKey;
+    private String apiSecret;
+
+    public CexioStreamingService(String apiUrl) {
+        super(apiUrl, Integer.MAX_VALUE);
+    }
+
+    @Override
+    protected String getChannelNameFromMessage(JsonNode message) throws IOException {
+        return null;
+    }
+
+    @Override
+    public String getSubscribeMessage(String channelName, Object... args) throws IOException {
+        return null;
+    }
+
+    @Override
+    public String getUnsubscribeMessage(String channelName) throws IOException {
+        return null;
+    }
+
+    @Override
+    public void messageHandler(String message) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode jsonNode;
+        try {
+            jsonNode = objectMapper.readTree(message);
+        } catch (IOException e) {
+            LOG.error("Error parsing incoming message to JSON: {}", message);
+            return;
+        }
+        handleMessage(jsonNode);
+    }
+
+    @Override
+    protected void handleMessage(JsonNode message) {
+        LOG.debug("Receiving message: {}", message);
+        JsonNode cexioMessage = message.get("e");
+
+        if (cexioMessage != null) {
+            switch (cexioMessage.textValue()) {
+                case CONNECTED:
+                    auth();
+                    break;
+                case AUTH:
+                    CexioWebSocketAuthResponse response =
+                            (CexioWebSocketAuthResponse)deserialize(message, CexioWebSocketAuthResponse.class);
+                    if ((response != null) && !response.isSuccess()) {
+                        LOG.error("Authentication error: {}", response.getData().getError());
+                    }
+                    break;
+                case PING:
+                    pong();
+                    break;
+                case ORDER:
```

Comment:
This part only parses incoming data, and don't provide they further. So this changes are good as prove of concept, but it not yet ready to merge.